bitnami/memcached/templates/hpa.yaml

```
@@ -0,0 +1,34 @@
+{{- if .Values.autoscaling.enabled }}
```

Only for HA architecture:
```suggestion
{{- if and .Values.Values.autoscaling.enabled (eq .Values.architecture "high-availability") }}
```